src/pages/TrueSight/hooks/useGetTrendingSoonData.ts

```diff
@@ -12,7 +12,7 @@ export interface TrueSightTokenData {
   name: string
   symbol: string
   rank: number | undefined // Trending soon only
-  platforms: Map<string, string>
+  platforms: Map<string, string> // ES6 Maps preserves the insertion order.
```

Comment:
Already included this description in #1162

Reply:
So We can close this PR right?